In [1]:
# Statistical Significance

from statsmodels.stats.proportion import proportions_ztest

conversions = [1261, 1228]
users = [12537, 12463]

z_stat, p_value = proportions_ztest(conversions, users)
print("Z-statistic:", round(z_stat,4))
print("P-value:", round(p_value,4))


Z-statistic: 0.5414
P-value: 0.5882


- The experiment showed a 2.09% decrease in conversion for the Treatment group compared to the Control group.

- However, the difference was not statistically significant (p-value = 0.5882).

- Therefore, there is insufficient evidence to conclude that the Treatment variant impacted user conversion.

### Recommendation:
- Continue with the Control experience and conduct additional experiments before rollout.